docs/xxjitservershareromclasses.md

```diff
@@ -34,7 +34,7 @@ This option enables the JITServer server to share cached ROM classes between JIT
 |`-XX:+JITServerShareROMClasses`           | Enable |                                                                                    |
```

Comment:
Would it be better to have this checked as enabled by default? Then the note could say something about needing to specify the `-XX:+JITServerShareROMClasses` explicitly to enable this feature if you've disabled the JITServer AOT cache at the server.